clients/src/test/java/org/apache/kafka/common/metrics/JmxReporterTest.java

```diff
@@ -35,6 +36,7 @@ public void testJmxRegistration() throws Exception {
         Metrics metrics = new Metrics();
         MBeanServer server = ManagementFactory.getPlatformMBeanServer();
         try {
+            JmxReporter reporter = new JmxReporter();
```

Comment:
looks like this is not passed to Metrics object. we can use reporter instance at below line.